### DISCUSSION TITLE
[E913] Corpus path can't be None.

### DISCUSSION THREAD

**Discussion by dsbyprateekg** [explosion/spaCy#9256] [Help: Coding & Implementations] [answered]
Hi,

I am using spacy 3.1.2 and trying to do custom NER training but getting Corpus-related errors. I could not find any solution to my issue so here I am attaching the required files and screenshots of the commands I used-

![spacy_ner_1](https://user-images.githubusercontent.com/30830541/134114176-4f6d2296-3333-48ca-b56f-ec45abbc463f.JPG)
![spacy_ner_2](https://user-images.githubusercontent.com/30830541/134114181-b91bbb20-00df-404f-b2d8-b46cfa615a64.JPG)
![spacy_ner_3](https://user-images.githubusercontent.com/30830541/134114183-afc092e8-5614-4fb5-b8a3-ea079124ed37.JPG)

Also, find below the JSON file I have used to convert into s[acy format-
[borrower_name_entity.txt](https://github.com/explosion/spaCy/files/7200357/borrower_name_entity.txt)

And the config file-
[config.txt](https://github.com/explosion/spaCy/files/7200365/config.txt)

Let me know what may be the issue since I am using train and dev.spacy files as well.

**Comment by svlandeg** [comment, marked as answer]
Hi!

Please copy/paste code (instead of taking screenshots) so others can run & replicate your issue and help you better.

Could you rerun the `debug` command again, but feed it the same override parameters for `paths.train` and `paths.dev`, and copy-paste the output here?
